common/storage/sql/sqlite3/storage.c

```diff
@@ -404,6 +419,40 @@ retcode_t iota_stor_transaction_update_solid_state(
   return ret;
 }
 
+retcode_t iota_stor_transactions_update_solid_state_true(
+    const connection_t* const conn, const hash243_set_t hashes) {
```

Comment:
Can't we encode the value to set as an argument? Having a _true (& _false) method sounds nasty - the overhead is also negligible

Reply:
sure